common/test/acceptance/tests/studio/base_studio_test.py

```diff
@@ -42,6 +42,8 @@ def log_in(self, user, is_staff=False):
         Log in as the user that created the course. The user will be given instructor access
         to the course and enrolled in it. By default the user will not have staff access unless
         is_staff is passed as True.
+        :param dict user: dictionary containing user data: {'username': ..., 'email': ..., 'password': ...}
```

Comment:
Nit: we use the Google documentation style for Python.

http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html